pkg/openshift-sdn/proxy.go

```diff
@@ -215,13 +216,25 @@ func (sdn *OpenShiftSDN) runProxy(waitChan chan<- bool) {
 	}
 
 	// Start up a metrics server if requested
+	if len(sdn.ProxyConfig.MetricsBindAddress) == 0 {
+		klog.Infof("Defaulting metrics bind path for debugging only")
+		sdn.ProxyConfig.MetricsBindAddress = "*:9665"
+	}
 	if len(sdn.ProxyConfig.MetricsBindAddress) > 0 {
 		mux := http.NewServeMux()
 		mux.HandleFunc("/proxyMode", func(w http.ResponseWriter, r *http.Request) {
 			fmt.Fprintf(w, "%s", sdn.ProxyConfig.Mode)
 		})
 		mux.Handle("/metrics", prometheus.Handler())
+
+		mux.HandleFunc("/debug/pprof/", pprof.Index)
```

Comment:
This port is open to the whole cluster - is there an information leak here?

Reply:
note the "DO NOT MERGE" on that commit :slightly_smiling_face:

Reply:
:) yeah I’m trying to look at what else might be broken.  Protobuf is the cheap easy win, but I’m seeing a lot of time in GC

Reply:
It's not (wasn't) on the PR :-)